randovania/games/am2r/json_data/Distribution Center.txt

```diff
@@ -1084,6 +1087,9 @@ Extra - map_name: rm_a5c26
           All of the following:
               # IBJ
               Infinite Bomb Jumping (Intermediate) and Can Use Bombs
+          All of the following:
```

Comment:
Can't make a comment on the actual line, so will make it here:
waterblob habitat should be changed to allow gravity+spider, and HJ, and maybe more things